src/keystore/InMemoryKeystore.ts

```diff
@@ -24,13 +24,21 @@ import {
   getKeyMaterial,
   topicDataToConversationReference,
 } from './utils'
-import { nsToDate } from '../utils'
+import { nsToDate, buildDirectMessageTopicV2 } from '../utils'
 import InviteStore from './InviteStore'
 import { Persistence } from './persistence'
 import LocalAuthenticator from '../authn/LocalAuthenticator'
 import { AccountLinkedRole } from '../crypto/Signature'
+import { sha256 } from 'ethers/lib/utils'
+import { hmacSha256Sign } from '../crypto/ecies'
+import { getRandomValues } from '../crypto/utils'
```

Comment:
nit: warnings about unused imports

Reply:
Fixed in https://github.com/xmtp/xmtp-js/pull/326/commits/0f091a41b5acc20f15e0c42edb34a21a7e7a1fa2.